src/commands/notify.yml

```diff
@@ -17,18 +17,19 @@ parameters:
   circle_token:
     description: >
       CircleCI token for fetching flaky tests via API.
+      If not specified, CIRCLE_TOKEN environment variable will be used.
       Please refer to this document how to generate it.
-      https://circleci.com/docs/managing-api-tokens
-    default: CIRCLE_TOKEN
-    type: env_var_name
+      https://circleci.com/docs/managing-api-tokens (requires a v2 token)
+    default: $CIRCLE_TOKEN
+    type: string
```

Comment:
I'm following how the [circleci slack orb](https://github.com/CircleCI-Public/slack-orb/blob/3e43e753919b007d681cb2a16b37cabe5a8eda83/src/commands/notify.yml#L57) works. 
From what I understand we're supporting passing either the value or the name of an env variable.
It took me a bit to fully understand that so I probably would opt for a simpler interface, but I didn't want to change anything too dramatic here.